application/src/components/view-orders/viewOrders.js

```diff
@@ -1,23 +1,93 @@
 import React, { Component } from 'react';
 import { Template } from '../../components';
 import { SERVER_IP } from '../../private';
+import MenuItemSelector from '../common/menuItemSelector';
+import { MENU_ITEMS, QUANTITY_VALUE_LIMITER } from '../../constants';
 import './viewOrders.css';
+import QuantitySelector from '../common/quantitySelector';
+
+const DELETE_ORDER_URL = `${SERVER_IP}/api/delete-order`;
+const EDIT_ORDER_URL = `${SERVER_IP}/api/edit-order`;
 
 class ViewOrders extends Component {
     state = {
-        orders: []
+        orders: [],
+        isLoading: true,
```

Comment:
**Note:** I should have used snake_case for the "isLoading" state property instead of camelCase.